Getting-Started/Developing-websites-with-Umbraco/index.md

```diff
@@ -0,0 +1,52 @@
+---
+meta.Title: "Developing websites with Umbraco"
+meta.Description: "This section shows you some beginner tools and information to get your started with Umbraco 8. From making a local installation to extending the backoffice."
+versionFrom: 8.0.0
+---
+# Developing websites with Umbraco
+
+Umbraco is built on top of a Microsoft MVC framework. You can build upon this technology to work alongside and extend the functionality in Umbraco. It is also designed to be pluggable so that you can replace key components with your own custom implementations if prefer.
+
+:::tip
+It is perfectly possible to build an Umbraco site without using Visual Studio and the techniques introduced on this page - see the [Creating websites with Umbraco](Creating-websites-with-Umbraco) section or take a look at our "no code/low code" option, [Umbraco Uno](Umbraco-Uno).
+:::
+
+This section is dedicated to introducing techniques that will help you get started with developing with an Umbraco site. You will find information on how to develop with the underlying framework of an Umbraco project as well as details on how you can extend the Umbraco backoffice to customize the editing experience.
+
+The concepts detailed in this section will go beyond standard out-of-the-box templating methodologies, and introduce some Umbraco specific terms and helper functionality, e.g. such as SurfaceControllers and management service APIs. All of which is technology that you can take advantage of when developing with Umbraco.
+
+You will also find information regarding Umbraco's underlying dependency injection framework.
+
+This will break into two sections: Extending the Umbraco backoffice and Developing custom websites.
+
+## [Extending the Umbraco backoffice](Extending-the-Umbraco-Backoffice)
+
+The Umbraco backoffice can be extended using AngularJS and C#. Customizing the Umbraco backoffice and editing experience includes creating your own Property Editors, Dashboards, and packages. You will also find information about how to customize things like Health Checks and the built-in search functionality.
+
+See [here](https://our.umbraco.com/documentation/Extending/) for a good place to start..
```

Comment:
I would suggest getting rid of this link here, as we should instead link to the section from within the "Extending the Umbraco backoffice" article.

What do you think?

```suggestion
```